k8s/kind/create_cluster.sh

```diff
@@ -80,8 +80,8 @@ ${FINAL_NODES}
 EOF
 
 if [[ $reg_network == "kind" ]]; then
-  docker network connect $reg_network "${reg_name}"
-  for node in $(kind get nodes); do
+  sudo docker network connect $reg_network "${reg_name}"
+  for node in $(sudo kind get nodes); do
```

Comment:
May check whether sudo is necessary for kind? For me it's not needed.

Reply:
I've updated the bootstrap script to get the user into docker group. This lets the user have permissions to access docker without sudo. Please review the latest update.